README.md

```diff
@@ -14,6 +14,9 @@ If you're using this module, and need help, you can:
 - Tweet to [@senecajs][],
 - Ask on the [Gitter][gitter-url].
 
+### Seneca compatibility
+Supports Seneca versions from **0.6.5** up to **2.0.0**
```

Comment:
incorrect readme, it should state the seneca versions tested in the travis.yml file